### PR TITLE
added hello

### PR DESCRIPTION
**Pull request by OmarJallouq**

### CODE REVIEW

File: skillbuddi/src/App.js [hunk 1]
```diff
@@ -2,12 +2,13 @@ import logo from './logo.svg';
...
 
 function App() {
+
   return (
     <div className="App">
       <header className="App-header">
         <img src={logo} className="App-logo" alt="logo" />
         <p>
-          Edit <code>src/App.js</code> and save to reload.
+          hello
```
Comment (OmarJallouq) [inline skillbuddi/src/App.js:11]: i dontb like this fix it

Comment (valentina-12345) [reply skillbuddi/src/App.js:11]: ok

**Comment by peczyk22** [comment]
ez its shit

File: skillbuddi/src/App.js (hunk 1 above)
Comment (FilipJanczak) [reply skillbuddi/src/App.js:11]: hiii guys!!!

Comment (peczyk22) [reply skillbuddi/src/App.js:11]: ez

**Review by peczyk22 — APPROVED** [review]

**Review by FilipJanczak — APPROVED** [review]
test

**Pull request closed without merging**